mm-mock-server/src/mocks/queries.ts

```diff
@@ -22,20 +22,10 @@ export function mockQueries(serverState: MockServerState) {
             return serverState.otherUsers.concat([serverState.loggedInUser]);
         },
         getAuthenticatedUser: () => {
-            if (!serverState.loggedIn) {
-                return null;
-            }
             return serverState.loggedInUser;
         },
-        getUserProfileInfo: () => {
```

Comment:
If we remove getUserProfileInfo, we should change kGetUserProfileInfo under lib/data/models/user/user_api.dart to querying getAuthenticatedUser?

Reply:
Yes, made this change in the latest updates